src/main/java/com/klarna/hiverunner/config/HiveRunnerConfig.java

```diff
@@ -93,12 +95,12 @@
     public static final String HIVECONF_SYSTEM_OVERRIDE_PREFIX = "hiveconf_";
 
     /**
-     * The shell's {@link CommandShellEmulation}.
+     * The shell's {@link CommandShellEmulator}.
      * 
      * Defaults to {@code HIVE_CLI}
      */
     public static final String COMMAND_SHELL_EMULATION_PROPERTY_NAME = "commandShellEmulation";
```

Comment:
OK, so as not to break backward compatibility I guess we want to leave this as "emulation". So yeah, I guess for public method names we should leave like this, for internal member variables they are "emulator"?